arangod/Agency/Agent.cpp

```diff
@@ -818,7 +822,7 @@ query_t Agent::lastAckedAgo() const {
       ret->add(i.first, VPackValue(
                  1.0e-2 * std::floor(
                    (i.first!=id() ?
-                    duration<double>(system_clock::now()-i.second).count()*100.0
+                    duration<double>(steady_clock::now()-i.second).count()*100.0
```

Comment:
Two digits of precision seems odd in light of senderTimeStamp being milliseconds (three digits of precision).  Is this intentional?  Would a comment in code stop folks from wondering?